File: backend/src/main/java/org/jdk/project/dto/config/RegionUpsertRequest.java
package org.jdk.project.dto.config;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class RegionUpsertRequest {
  private Long areaId;
  private Long loungeId;
  private String regionName;
  private String coordinate;
  private Integer maxCapacity;
  private String isGuide;
  private String isShow;
  private Integer enable;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/RegionUpsertRequest.kt
package org.jdk.project.dto.config

data class RegionUpsertRequest(
    var areaId: Long? = null,
    var loungeId: Long? = null,
    var regionName: String? = null,
    var coordinate: String? = null,
    var maxCapacity: Int? = null,
    var isGuide: String? = null,
    var isShow: String? = null,
    var enable: Int? = null,
    var remark: String? = null,
)
